### PR TITLE
More analytics/kb

### PR DESCRIPTION
**Pull request by kj13ennett**

### CODE REVIEW

File: pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLPhoto.java [hunk 1]
```diff
@@ -277,6 +277,26 @@ public Integer sourceIconImage() {
...
+     * Analytics methods
+     */
+
+    public void actionClicked(String actionLink, Context context) {
+        PXLClient pxlClient = PXLClient.getInstance(context);
+        JSONObject body = new JSONObject();
+
+        try{
+            body.put("album_id",  Integer.parseInt(this.album.id));
+            body.put("alum_photo_id", Integer.parseInt(this.albumPhotoId));
```
Comment (firetix) [inline pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLPhoto.java]: mispelling album_photo_id

**Comment by firetix** [comment]
Can you update the read.me as well, Thanks :)

File: pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLAlbum.java [hunk 2]
```diff
@@ -214,4 +218,34 @@ public boolean openedWidget() {
...
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+
+        try{
+            body.put("album_id",  Integer.parseInt(this.id));
+            body.put("per_page", this.perPage);
+            body.put("page", this.page);
+            body.put("photos", stringBuilder.toString());
```
Comment (jchen123) [inline pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLAlbum.java]: @firetix  if kb adds this to android it needs to get added to ios too

Comment (firetix) [reply pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLAlbum.java]: I did https://github.com/pixlee/ios-sdk-carthage/pull/7/files#diff-3a24009da7c676cdd9b02eee9de5f1ddR289

File: pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLPhoto.java (hunk 1 above)
Comment (jchen123) [reply pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLPhoto.java]: @kj13ennett dont let this slip lol

**Comment by kj13ennett** [comment]
cURL examples for ya:
Action Clicked: `curl -X POST \
  https://inbound-analytics.pixlee.com/events/actionClicked \
  -H 'Content-Type: application/json' \
  -H 'cache-control: no-cache' \
  -d '{"album_id":2969283,"album_photo_id":332748379,"action_link_url":"https:\/\/ca.puma.com\/en\/ca\/pd\/clyde-court-core-basketball-shoes\/191712.html","API_KEY":"196i8ZzIAhKU8dO2kDe","uid":"d5dffcccf5db0134","platform":"android"}'`

loadMore: `curl -X POST \
  https://inbound-analytics.pixlee.com/events/loadMore \
  -H 'Content-Type: application/json' \
  -H 'cache-control: no-cache' \
  -d '{"album_id":4503434,"per_page":20,"page":2,"photos":"297773220,294777131,292741086,289691280,285563952,282370551,280753281,277286185","API_KEY":"196i8ZzIAhKU8dO2kDe","uid":"d5dffcccf5db0134","platform":"android"}'`

**Comment by jchen123** [comment]
@kj13ennett lgtm

**Comment by firetix** [comment]
lgtm

File: README.md [hunk 3]
```diff
@@ -83,6 +86,15 @@ To fire an opened ligtbox event, simply call the `openedLightbox` method of the
 photo.openedLightbox(context);
 ```
 
+#### Action Clicked
+To fire an action clicked event, simply call the `actionClicked` method of the PXLPhoto that the action click is being driven from, and an "Action Clicked" event will be fired containing all of the necessary analytics information.
```
Comment (firetix) [inline README.md]: Can you adjust the text here, they need to pass the product link of the product clicked for example if a photo has multiple products.

Comment (kj13ennett) [reply README.md]: yes, great catch!

**Comment by firetix** [comment]
lgtm! go! go!

**Pull request merged**